docs/rst/tutorials/core/qos/mutable/mutable.rst

```diff
@@ -0,0 +1,296 @@
+.. _tutorials_qos_mutable_mutable:
+
+Change mutable QoS through get native API
+=========================================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+Background
```

Comment:
I think I'm lacking a visual illustration of what the tutorial shows here

Reply:
Could you please explain or show an example?